pysidt/extensions.py

```diff
@@ -24,7 +24,7 @@ def split_mols(data, newgrp):
 
     return new, comp
     
-def get_extension_edge(parent, n_splits, iter_max=np.inf, iter_item_cap=np.inf):
+def get_extension_edge(parent, n_splits, iter_max=np.inf, iter_item_cap=np.inf, r=None, r_bonds=[1,2,3,1.5,4], r_un=[0,1,2,3]):
```

Comment:
I think generally we don't want to use mutable objects as defaults? Also I think I'm a big fan of being explicit about the type of the input, and maybe we want to adapt that style.

Reply:
Can you also add docstring for the inputs